winml/lib/Api/ImageFeatureDescriptor.cpp

```diff
@@ -32,6 +32,28 @@ ImageFeatureDescriptor::ImageFeatureDescriptor(
                                               color_space_gamma_(color_space_gamma) {
 }
 
+ImageFeatureDescriptor::ImageFeatureDescriptor(
+        hstring const& Name,
```

Comment:
nit: the naming convention of these parameters don't match what's on the other constructor of ImageFeatureDescriptor.

Reply:
right, these got copied from the auto gen code.   we should decide how we want to do coding conventions, but my rough idea is something that follows along lineage.   so if autogen code does something a certain way, we stick that that.  if ort has another way, we stick with that.  etc.etc..  based on lineage.

---
In reply to: [344926984](https://github.com/microsoft/onnxruntime/pull/2368#discussion_r344926984) [](ancestors = 344926984)